platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/event/GossipEventTest.java

```diff
@@ -51,6 +56,21 @@ void test() throws IOException, ConstructableRegistryException {
         assertEquals(gossipEvent, copy, "deserialized version should be the same");
     }
 
+    @Test
+    @DisplayName("Deserialize prior version of event")
+    void deserializePriorVersion() throws IOException, ConstructableRegistryException {
```

Comment:
should we also check the signature if its valid?

Reply:
The most important thing is that the bytes on the wire are preserved.  I no longer know which public key produced the event for the purpose of validation.  I've extended the unit test to check the bytes serialized are identical to the file contents.